app/mattermost.android.js

```diff
@@ -0,0 +1,436 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import 'babel-polyfill';
```

Comment:
@jarredwitt This may be hard to follow but this is the basic anatomy of `mattemost.android.js`

1. load heavy modules in a lazy manner
2. initializeModules function ready to be called after Channel has been launched.
   * These modules do not depend on cold startup, so they can be initialized after the Channel has rendered
3. contain global listeners/callbacks in this file so it can maintain reference throught the app's existence `mattermost.js` class
4. start `Entry` screen, notice not much has been initialized

After this we do the same thing wait for `redux rehydration`